62469: Expand note about modifying plugin jails a little (#622)
freenas/freenas-docs: userguide/plugins.rst

:menuselection:`Plugins --> Installed`
page as shown in :numref:`Figure %s <view_installed_plugins_fig>`.

.. note:: Plugins are also added to
   :menuselection:`Jails`

>    as a *pluginv2* jail. This jail type is adjustable just like a

Reviewer: The jail type is adjustable?  I think this means "This type of jail is editable like a"
```suggestion
   as a *pluginv2* jail. This type of jail is editable like a
```

>    standard jail, except the *UUID* cannot be altered.

Reviewer: ```suggestion
   standard jail, but the *UUID* cannot be altered.
```

>    See :ref:`Managing Jails` for more details about modifying plugin

Reviewer: ```suggestion
   See :ref:`Managing Jails` for more details about modifying
```
